### PR TITLE
To test modify resource options in PV Pool Backingstore

### PR DESCRIPTION
**Pull request by mashetty330**
Signed-off-by: mashetty <mashetty@redhat.com>

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2064599

### CODE REVIEW

**Review by parikshithb — COMMENTED** [review]
Looks good. I suggest to create pv bs backed bucket and then write/read from bucket after changing the resources.

**Comment by mashetty330** [comment]
> Looks good. I suggest to create pv bs backed bucket and then write/read from bucket after changing the resources.

@parikshithb ack. but why do we need to create bucket and test write/read?

**Comment by parikshithb** [comment]
> > Looks good. I suggest to create pv bs backed bucket and then write/read from bucket after changing the resources.
> 
> @parikshithb ack. but why do we need to create bucket and test write/read?

To validate whether changing resources will not affect IOs.

**Comment by mashetty330** [comment]
> > > Looks good. I suggest to create pv bs backed bucket and then write/read from bucket after changing the resources.
> > 
> > 
> > @parikshithb ack. but why do we need to create bucket and test write/read?
> 
> To validate whether changing resources will not affect IOs.

@parikshithb I tried to populate the bucket using sync. But its failing with the issues mentioned here https://github.com/red-hat-storage/ocs-ci/issues/3323 
This is the line of code:
`        objects_list = write_random_objects_in_pod(io_pod=awscli_pod_session,
                                                   file_dir=file_dir,
                                                   amount=5,
                                                   pattern="test-object-",)
        sync_object_directory(podobj=awscli_pod_session, src=file_dir, target=f"s3://{bucket_name}", s3_obj=OBC(bucket_name))
`

**Review by parikshithb — APPROVED** [review]

File: tests/manage/mcg/test_pv_pool.py [hunk 1]
```diff
@@ -120,3 +130,93 @@ def test_pv_scale_out(self, backingstore_factory):
             backingstore_dict["spec"]["pvPool"]["numVolumes"] == pv_backingstore.vol_num
         ), "Scale out PV Pool failed. "
         logger.info("Scale out was successful")
+
+    @pytest.mark.polarion_id("OCS-3932")
+    @tier2
+    @bugzilla("2064599")
+    def test_modify_resource(
```
Comment (MeridianExplorer) [inline tests/manage/mcg/test_pv_pool.py:137]:
```suggestion
    def test_pvpool_cpu_and_memory_modifications(
```

**Review by MeridianExplorer — COMMENTED** [review]
Minor comments

**Review by MeridianExplorer — APPROVED** [review]

**Review by PrasadDesala — APPROVED** [review]

**Pull request merged**